release-repo-scripts/bump_dependency.bash

```diff
@@ -455,15 +455,10 @@ for ((i = 0; i < "${#LIBRARIES[@]}"; i++)); do
     DEP_VER=${VERSIONS[$j]}
     DEP_PREV_VER="$((${DEP_VER}-1))"
 
-    # Rule: *plugin2 -> *plugin3
-    # Replace lines like: "find_package(ignition-cmake2)"
-    #               with: "find_package(ignition-cmake3)"
-    find . -type f ! -name 'Changelog.md' ! -name 'Migration.md' -print0 | xargs -0 sed -i "s ${DEP_LIB}${DEP_PREV_VER} ${DEP_LIB}${DEP_VER} g"
-
-    # Rule: IGN_PLUGIN_VER 2 -> IGN_PLUGIN_VER 3
+    # Rule: IGN_<LIB>_VER <N> -> IGN_<LIB>_VER ${ignition-<lib><N>_VERSION_MAJOR}
     # Replace lines like: "set(IGN_PLUGIN_VER 2)"
-    #               with: "set(IGN_PLUGIN_VER 3)"
-    find . -type f ! -name 'Changelog.md' ! -name 'Migration.md' -print0 | xargs -0 sed -i "s@IGN_${DEP_LIB}_VER ${DEP_PREV_VER}@\UIGN_${DEP_LIB}_VER ${DEP_VER}@ig"
+    #               with: "set(IGN_PLUGIN_VER ${ignition-plugin3_VERSION_MAJOR})"
+    find . -type f -name 'CMakeLists.txt' -print0 | xargs -0 sed -i "s@IGN_${DEP_LIB}_VER ${DEP_PREV_VER}@\UIGN_${DEP_LIB}_VER \L\$\{ignition-${DEP_LIB}${DEP_VER}_\UVERSION_MAJOR\}@ig"
```

Comment:
I would remove this block and rely on updating the examples to use `*_VERSION_MAJOR` variables. I'll open an example PR to start

Reply:
https://github.com/ignitionrobotics/ign-rendering/pull/607

Reply:
https://github.com/ignitionrobotics/ign-math/pull/408

Reply:
I'm going to stop now; I think there may be a few more examples folders to check

Reply:
Thanks for helping with them! You got all of them except these: https://github.com/ignitionrobotics/ign-gazebo/pull/1442

:fire: